### PR TITLE
[DATAHELP-2984] Log more on deepcopy failure

### PR DESCRIPTION
**Pull request by youngyjd**
This one is similar to https://github.com/lyft/incubator-airflow/pull/174/files but the request I got enters into the other condition, hence adding it as well.
https://jira.lyft.net/browse/DATAHELP-2984

### CODE REVIEW

**Comment by youngyjd** [comment]
@jinhyukchang @astahlman @feng-tao

File: airflow/models.py [hunk 1]
```diff
@@ -2355,10 +2355,17 @@ def __deepcopy__(self, memo):
...
-                                      'key: {k} ,value: {v}'.format(k=k, v=v))
+                                      'key: {k}, key type: {tk}, value: {v}, value type: {tv}'.format(
+                                        k=k, tk=type(k), v=v, tv=type(v)))
                     raise e
             else:
-                setattr(result, k, copy.copy(v))
+                try:
+                    setattr(result, k, copy.copy(v))
+                except Exception as e:
+                    logging.exception('Failed deepcopy on member variable '
```
Comment (andelink) [inline airflow/models.py]: Nit: should this read *Failed **shallow copy** on member variable* ?

File: airflow/models.py [hunk 2]
```diff
@@ -2355,10 +2355,17 @@ def __deepcopy__(self, memo):
                     setattr(result, k, copy.deepcopy(v, memo))
                 except Exception as e:
                     logging.exception('Failed deepcopy on member variable '
-                                      'key: {k} ,value: {v}'.format(k=k, v=v))
+                                      'key: {k}, key type: {tk}, value: {v}, value type: {tv}'.format(
+                                        k=k, tk=type(k), v=v, tv=type(v)))
                     raise e
             else:
-                setattr(result, k, copy.copy(v))
+                try:
```
Comment (jinhyukchang) [inline airflow/models.py]: I wonder if we can just have one try...except block for it seems very similar. What do you think @youngyjd ?

Comment (youngyjd) [reply airflow/models.py]: sure

File: airflow/models.py (hunk 1 above)
Comment (youngyjd) [reply airflow/models.py]: What does it mean?

**Pull request merged**